### ISSUE TITLE
Multipanel plot return type

### ISSUE THREAD

**Issue by dan-knight** [uclahs-cds/package-BoutrosLab-plotting-general#172]
@dan-knight Pulling you in for a BPG question:
It seems that `create.multipanelplot` does not return a plot object if a filename is specified (it just writes the plot to the file). Is that expected behavior? Also, when an object is returned, it seems to be of class "multipanel" - is that something that was made up for BPG or is that an actual lattice-related class?

_Originally posted by @alkaZeltser in https://github.com/uclahs-cds/package-ApplyPolygenicScore/pull/24#discussion_r1542076482_

**Comment by jarbet** [comment]
> I think that should be changed to include the other trellis/grid classnames (adding `multipanel`) rather than setting it to only `multipanel`.

Agreed!  There is a bug where you can't immediately input a plot made by `create.multipanelplot` into another `create.multipanelplot` call.

The temporary solution @stefaneng came up with is to just redefine the class:

```
class(multipanel.plot) <- c('multipanel', 'frame', 'gTree', 'grob', 'gDesc');
```

Then the above `multipanel.plot` can be input into another `create.multipanelplot` call.

**Comment by pboutros** [comment]
I have no memory of this either way, but suggest tracing the initial commit history of `create.multipanelplot` on svn to see if there's anything useful in there
